packages/roosterjs-editor-plugins/lib/plugins/ImageEdit/editInfoUtils/applyChange.ts

```diff
@@ -68,14 +69,36 @@ export default function applyChange(
     // Write back the change to image, and set its new size
     const { targetWidth, targetHeight } = getGeneratedImageSize(editInfo);
     image.src = newSrc;
-    image.style.width = targetWidth + 'px';
-    image.style.height = targetHeight + 'px';
-    image.width = targetWidth;
-    image.height = targetHeight;
+    setImageSize(image, wasResized, targetWidth, targetHeight);
 
     return (
         srcChanged ||
         editInfo.widthPx != initEditInfo.widthPx ||
         editInfo.heightPx != initEditInfo.heightPx
     );
 }
+
+/**
+ * @param img The current image.
+ * @param wasResized the current resize state of the image
+ */
+function setImageSize(
+    image: HTMLImageElement,
+    wasResized: boolean,
+    targetWidth: number,
+    targetHeight: number
+) {
+    if (wasResized) {
+        image.style.maxWidth = 'initial';
+        image.width = targetWidth;
+        image.height = targetHeight;
+        image.style.width = targetWidth + 'px';
+        image.style.height = targetHeight + 'px';
+    } else {
+        image.style.maxWidth = '100%';
+        image.style.height = 'initial';
+        image.removeAttribute('height');
+        image.removeAttribute('width');
+        image.style.width = '';
```

Comment:
why set style.height to "initial" but set style.width to "" ? They should be the same.